tests/skimage/metrics/test_structural_similarity.py

```
@@ -143,12 +146,13 @@ def test_structural_similarity_multichannel(channel_axis):
 def test_structural_similarity_nD(dtype):
     # test 1D through 4D on small random arrays
     N = 10
+    rng = np.random.RandomState(978242013)
     for ndim in range(1, 5):
```

Ruff (0.15.5)

[warning] Loop control variable `ndim` not used within loop body

Rename unused `ndim` to `_ndim`

(B007)